microservices/Order.py

```
@@ -0,0 +1,132 @@
+        # return "This page will show all orders"
+        return render_template('orders.html', orders = orders, title = "orders")
+
+
+@app.route('/order/new/', methods = ['GET', 'POST'])
+# Method to create neworder
+def neworder():
+    # Check to see if a user is logged in.
+    if 'username' not in login_session:
+        return redirect('/login')
+
+    # If user is logged in then access neworder page
+    if request.method == 'POST':
+        # Store order input from form into neworder
+        neworder = order(
+            name = request.form['name'].strip(),
+            street = request.form['street'].strip(),
+            city = request.form['city'].strip(),
+            state = request.form['state'].strip(),
+            zipCode = request.form['zipCode'].strip(),
+            phone = request.form['phone'].strip(),
+            email = request.form['email'].strip(),
+            website = request.form['website'].strip(),
+            cuisine = request.form['cuisine'].strip(),
+            description = request.form['description'].strip(),
+            delivery = request.form['delivery'].strip(),
+            user_id = login_session['user_id']
+        )
+        # add neworder item to db stage
+        session.add(neworder)        # commit neworder item to db
+        session.commit()
+
+        # flash message to indicate success
+        flash("New order: " + neworder.name + " ==> Created!")
+        # redirect user to updated list of orders
+        return redirect(url_for('showorders'))
+    else:
+        return render_template('neworder.html', title = "New order Input")
+
+# Edit a order
+#########################
+@app.route('/order/order id/edit/', methods = ['GET', 'POST'])
+def editorder(order_id):
+
+    # query db by order_id and assign to order variable
+    order = session.query(order).filter_by(id = order_id).one()
+
+    # Check to see if a user is logged in.
+    if 'username' not in login_session:
+        return redirect('/login')

```

this should return a JSON error